index.js

```diff
@@ -60,13 +60,35 @@ animalToRemove.remove()
 
 /***** Deliverable 1 *****/
 function toggleColor(element) {
-    if (element.style.color === "green") {
+    if (element.style.color === "red") {
         element.style.color = "black"
     } else {
-        element.style.color = "green"
+        element.style.color = "red"
     }
 }
 
+header.addEventListener('click', function(event){
+    toggleColor(event.target)
+
+})
+
 /***** Deliverable 2 *****/
+const button = document.querySelector('button.like-button')
+button.addEventListener('click', function() {
+    const nowlikes = parseInt(likes.textContent)
+    likes.textContent = `${nowlikes + 1} Likes`
+})
```

Comment:
First, you find the like button element on the page. Then, you add an event listener to the like button passing in the event type and callback function as arguments. Great work!